sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -0,0 +1,188 @@
+-- Turns on ANSI mode
+SET spark.sql.parser.ansi.enabled=true;
```

Comment:
How about moving these kinds of ANSI-related tests into a new dir `sql-tests/inputs/ansi/`?

Reply:
SGTM

Reply:
ok